File: leetcode/Array/217. Contains Duplicate/Solution.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Arrays.sort(nums);
        boolean isDuplicate = false;
        for(int i=1; i< nums.length; i++) {
            if(nums[i-1]==nums[i]) isDuplicate = true;
        }
        return isDuplicate;
    }
}

File: leetcode/Array/217. Contains Duplicate/Solution.kt
class Solution {
    fun containsDuplicate(nums: IntArray): Boolean {
        nums.sortDescending()
        var returnValue = false
        for(i in 1..<nums.size) {
            if(nums[i] == nums[i - 1]) {
                returnValue = true
            }
        }
        return returnValue
    }
}
